src/Optical/OpticalSystem.jl

```diff
@@ -374,7 +374,12 @@ struct AxisymmetricOpticalSystem{T,C<:CSGOpticalSystem{T}} <: AbstractOpticalSys
             elseif surface_type == "Aspheric"
                 semidiameter = convert(T, max(get_front_back_property(prescription, i, "SemiDiameter", zero(T))...))
```

Comment:
don't think a call to convert is necessary here; pretty sure it is called implicitly since semidiameter is declared of type T.

Reply:
Agreed, this is much better. I was using `convert(T, x)` to replace several usages of `T(x)` which were being highlighted as "possible method call errors" by VSCode.

I'm adding a commit that makes better use of type declarations. This also helps with readability.